armi/reactor/components/complexShapes.py

```diff
@@ -140,6 +137,69 @@ def getCircleInnerDiameter(self, Tc=None, cold=False):
         return self.getDimension("holeOP", Tc, cold)
 
 
+class FilletedHexagon(basicShapes.Hexagon):
+    """Hexagon with rounded corners.
+
+    This shape has a hexagonal hole with rounded corners cut out of the center of it. By default,
```

Comment:
This description is really confusing to me. Is this trying to model a hexagon where the corners are filleted on both the inner and outer surface?

Reply:
Yup!